packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -114,6 +114,16 @@ The following table provides a list of all available email templates in an Okta
 | Reassigned Review | Access certification campaign: (`campaignName`) | `IGAReviewerReassigned` | | Sent to reviewers when they are assigned new reviews by an admin or a different reviewer. |
 | Idp MyAccount Email Change Confirmation | Confirm email address change | `MyAccountChangeConfirmation` |  | Sent to users who try to verify an email address using MyAccount APIs. The users must enter the provided code to confirm the change. |
 
+### Use Branding variables
+
+After you create a theme for your org, you can use branding variables in your custom email templates by selecting the `FULL_THEME` variant for your emails. If you delete any of these optional variables from the email code, it will disconnect the theme object properties from the email template.
+
+| Variable | Asset Type |
+|-------------------|--------------------|
+| ${brand.theme.logo}| URL |
```

Comment:
@gormanho-okta I've been seeing that there is also just a `$brand` variable listed with optionalReason = 'branding' in the API response. Should this be listed here? Or should it not be considered as a branding variable?

Reply:
Actually, the entire list of optional branding variables is:
```
      "brand",
      "brand.brandId",
      "brand.theme.primaryColorContrast",
      "brand.theme.favicon",
      "brand.theme.backgroundImage",
      "brand.theme.primaryColor",
      "brand.theme.themeId",
      "brand.theme.logo",
      "brand.theme.secondaryColorContrast",
      "brand.theme.secondaryColor"
```
But I don't think we need to list them all here. As far as I can tell, the 3 already listed are the only ones being used in the default templates and provide all the theming the user sees out of the box. Do we already have a comprehensive list of these branding variables elsewhere, @brentschaus-okta ?

Reply:
I'm going to keep it to the 3 I documented because those are default in emails, the others in this list don't work with emails today.